operator/controllers/lifecycle/keptnappversion/reconcile_workloadsstate.go

```diff
@@ -20,18 +19,30 @@ func (r *KeptnAppVersionReconciler) reconcileWorkloads(ctx context.Context, appV
 		LongName:  "Reconcile Workloads",
 	}
 
-	var newStatus []klcv1alpha3.WorkloadStatus
+	workloadInstanceList, err := r.getWorkloadInstanceList(ctx, appVersion.Namespace, appVersion.Spec.AppName)
+	if err != nil {
+		r.Log.Error(err, "Could not get workloads")
```

Comment:
let's add the name of the appversion to the log entry to give the log message more context